build-tools/src/main/resources/build-tools/spotbugs-filter.xml

```diff
@@ -90,4 +90,14 @@ limitations under the License.
     <Class name="org.apache.knox.gateway.audit.log4j.correlation.Log4jCorrelationService" />
     <Bug pattern="OBJECT_DESERIALIZATION" />
   </Match>
+
+  <Match>
+    <Class name="org.apache.knox.gateway.services.security.impl.ConfigurableEncryptor" />
+    <Bug pattern="DMI_RANDOM_USED_ONLY_ONCE" />
+  </Match>
+
+  <Match>
+    <Class name="org.apache.knox.gateway.services.registry.impl.DefaultServiceRegistryService" />
+    <Bug pattern="DMI_RANDOM_USED_ONLY_ONCE" />
```

Comment:
I think this is also a real finding - https://github.com/apache/knox/blob/master/gateway-server/src/main/java/org/apache/knox/gateway/services/registry/impl/DefaultServiceRegistryService.java#L74